chaosaws/cloudwatch/probes.py

```diff
@@ -142,6 +162,10 @@ def get_metric_data(namespace: str, metric_name: str, dimension_name: str,
 
     if unit:
         args['MetricDataQueries'][0]['MetricStat']['Unit'] = unit
+    if dimensions:
+        args['MetricDataQueries'][0]['MetricStat']['Metric']['Dimensions'] = dimensions
+    if dimension_name and dimension_value:
+        args['MetricDataQueries'][0]['MetricStat']['Metric']['Dimensions'] = [{'Name': dimension_name, 'Value': dimension_value}]
```

Comment:
@Lawouach what are your thoughts on the precedence of variables here?
Should it be `else` rather than `if`'s?

If dimensions are there, don't then replace them with the `dimension_name` and `dimension_value` list?

Reply:
I think the ordering is fine here. But I'd have a `elif` rather than two `if`.